### PR TITLE
MNTOR-3026: Remove some unused feature flags

### PR DESCRIPTION
**Pull request by mansaj**
# References: 
Jira: MNTOR-3026

# Description
Remove some unused feature flags
* FreeBrokerScan
* PremiumBrokerRemoval
* FalseDoorTest
* HibpBreachNotifications
* FxaUidTelemetry
* RebrandAnnouncement
* MonitorAccountDeletion
* LastScanDateBadge

# Checklist (Definition of Done)
- [x] Localization strings (if needed) have been added.
- [x] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
- [x] I've added or updated the relevant sections in readme and/or code comments
- [x] I've added a unit test to test for potential regressions of this bug.
- [x] All acceptance criteria are met.
- [x] Jira ticket has been updated (if needed) to match changes made during the development process.
- [x] Jira ticket has been updated (if needed) with suggestions for QA when this PR is deployed to stage.

### CODE REVIEW

File: src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/View.tsx [hunk 1]
```diff
@@ -130,120 +130,99 @@ export const SettingsView = (props: Props) => {
...
-                          )}
-                        </p>
-                        <p>
-                          {l10n.getString(
-                            "settings-delete-monitor-plus-account-dialog-lead-p2-2",
-                          )}
-                        </p>
-                      </div>
-                      <DeleteAccountButton variant="primary">
+
```
Comment (mansaj) [inline src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/View.tsx:133]: @Vinnl got rid of the first condition

File: locales/en/settings.ftl [hunk 2]
```diff
@@ -49,12 +49,6 @@ settings-email-number-of-breaches-info =
        *[other] Appears in { $breachCount } known breaches.
   }
```
Comment (mansaj) [inline locales/en/settings.ftl:51]: @Vinnl deleted ftl strings

**Comment by mansaj** [comment]
Ran against e2e full suite and passed: https://github.com/mozilla/blurts-server/actions/runs/9215267991

**Comment by mansaj** [comment]
@Vinnl not sure why that one unrelated file's coverage decreased `useLocalDismissal`

**Review by flodolo — APPROVED** [review]

**Review by Vinnl — APPROVED** [review]
Thanks for the inline comments, those were helpful in reviewing, and I love to see the cleanup.

As for the decreased coverage: that's because of the removal of `<RebrandAnnouncement>`, which exercised that line of code. I suppose we can add an ignore comment for that line stating that it is unused at the moment - it will probably be useful again for future banners.

File: src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/View.tsx [hunk 3]
```diff
@@ -130,120 +130,99 @@ export const SettingsView = (props: Props) => {
             </>
           )}
           <hr />
-          {!props.enabledFeatureFlags.includes("MonitorAccountDeletion") ? (
-            <div className={styles.deactivateSection}>
```
Comment (Vinnl) [inline src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/View.tsx:134]: Oh, looks like I forgot my comment - for an extra thorough cleanup, you could also remove the reference to this class in https://github.com/mozilla/blurts-server/blob/52ea336a9da4c5ef95aa5e6384b9e95e0e520567/src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/View.module.scss#L58

**Comment by mansaj** [comment]
> Thanks for the inline comments, those were helpful in reviewing, and I love to see the cleanup.
> 
> As for the decreased coverage: that's because of the removal of `<RebrandAnnouncement>`, which exercised that line of code. I suppose we can add an ignore comment for that line stating that it is unused at the moment - it will probably be useful again for future banners.

Ah that's good to know. I couldn't reproduce it locally so I tried a few places and couldn't get the coverage.

**Review by rhelmer — APPROVED** [review]
Thanks!

File: src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/View.tsx (hunk 3 above)
Comment (mansaj) [reply src/app/(proper_react)/(redesign)/(authenticated)/user/(dashboard)/settings/View.tsx:134]: [chore: remove ref for deactivateSection](https://github.com/mozilla/blurts-server/pull/4586/commits/a1fdf1c795ea5b6c4403e346ef2b392fecf8aa95)

**Pull request merged**